Cargo.toml

```diff
@@ -1,3 +1,8 @@
 [workspace]
 resolver = "2"
 members = ["abci", "proto-compiler", "proto"]
+
+[workspace.package]
+
+rust-version = "1.76"
```

Comment:
1.80?

Reply:
This is "minimal" version supported by the crate, and it supports 1.76.

Bumping this will be a breaking change.

But I have changed the Dockerfiles to ensure everything builds on 1.76.

Reply:
Actually, I found out that it doesn't build cleanly on 1.76, updating to 1.80 as you suggested.